README.md

```diff
@@ -92,21 +103,28 @@ The body, headers, etc. for this response will be taken from the API description
 Requests to operations which expect a request body will be validated, for example: a POST with JSON.
 
 ```bash
-http --json POST http://127.0.0.1:4010/pets name=Stowford
+curl -s -D "/dev/stderr" -H "content-type: application/json" -d '{"name":"Stowford"}' http://127.0.0.1:4010/pets?__code==405 | json_pp
 ```
 
-This will generate an error:
+This will generate an error, conforming the [application/problem+json][rfc7807] specification:
 
 ```
-Here is the original validation result instead: [{"path":["body"],"name":"required","summary":"should have required property 'photoUrls'","message":"should have required property 'photoUrls'","severity":"error"}]
+HTTP/1.1 422 Unprocessable Entity
+content-type: application/problem+json
+content-length: 279
+Date: Wed, 08 May 2019 19:22:27 GMT
+Connection: keep-alive
+
+{
+   "title" : "Invalid request body payload",
+   "detail" : "Your request body is not valid: [{\"path\":[\"body\"],\"code\":\"required\",\"message\":\"should have required property 'photoUrls'\",\"severity\":0}]",
+   "status" : 422,
+   "name" : "https://stoplight.io/prism/errors#UNPROCESSABLE_ENTITY"
+}
 ```
 
 This error shows the request is missing the `photoUrls` property:
```

Comment:
We removed the example but left the intro in place.

Reply:
Hmm no, it's explaining the error above. The request just done above is missing that property. Maybe we can reword/move this somewhere else? @philsturgeon

Reply:
```suggestion
This error shows the request is missing a required property `photoUrls` from the HTTP request body.
```

Reply:
That'll do it.